Sources/TuistKit/Services/TestService.swift

```diff
@@ -204,8 +204,11 @@ final class TestService {
             throw TestServiceError.schemeWithoutTestableTargets(scheme: scheme.name)
         }
 
+        let platform = try buildableTarget.target.servicePlatform
```

Comment:
I find `servicePlatform` not very representative of what the platform does but I can't come up with a better name 😬. If you could describe what the platform returned by this getter is, how would you describe it?